docs/workflow/testing/libraries/filtering-tests.md

```diff
@@ -176,3 +234,67 @@ dotnet build <csproj_file> /t:Test /p:TargetOS=windows
 ```sh
 dotnet build <csproj_file> /t:Test /p:TargetOS=OSX /p:WithCategories="OuterLoop;failing""
 ```
+
+## SkipOnCoreClrAttribute
+This attribute is used to disable a test under specific conditions, only when run with CoreCLR (it doesn't affect tests run with Mono). Typically, this is when there is a failure in a particular test run configuration, such as under GCStress or JitStress.
+
+This attribute can be applied either to a test class (will disable all the tests in that class) or to a test method. It allows multiple usages on the same member.
+
+This attribute is defined [here](https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.XUnitExtensions/src/Attributes/SkipOnCoreClrAttribute.cs).
+
+**Disable for all platforms and all target frameworks:**
+
+```cs
+[SkipOnCoreClr(string reason)]
+```
+Example:
+```cs
+[SkipOnCoreClr("CoreCLR does track thread specific JIT information")]
+```
+
+**Disable for specific platform:**
+
+```cs
+[SkipOnCoreClr(string reason, TestPlatforms testPlatforms)]
+```
+Example:
+```cs
+[SkipOnCoreClr("Long running tests: https://github.com/dotnet/runtime/issues/11980", TestPlatforms.Linux)
+```
+
+**Disable for specific test mode:**
```

Comment:
Is there a way to disable a test for any "slow" mode? that might be a debug runtime, or some stress mode. Not sure whether that's a scenario.

Reply:
Ah yes you can have multiple such attributes

        [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/67886", ~RuntimeConfiguration.Release)]
        [SkipOnCoreClr("https://github.com/dotnet/runtime/issues/67886", RuntimeTestModes.JitMinOpts)]